docs/getting_started/basic/zero-shot-prediction/zero-shot-classification.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>"it is now possible to perform zero-shot learning" Maybe provide a link to zero-shot learning </p><p>Link to ImageNet-O dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>